ngraph/frontend/onnx_import/src/core/graph.cpp

```diff
@@ -95,9 +95,14 @@ namespace ngraph
                     }
                     catch (const ngraph::ngraph_error& exc)
                     {
-                        NGRAPH_WARN << "Could not create an nGraph Constant for initializer '"
-                                    << initializer_tensor.name() << "'. Detailed error:\n"
-                                    << exc.what();
+                        NGRAPH_WARN
+                            << "\nCould not create an nGraph Constant for initializer '"
+                            << initializer_tensor.name() << "'. \n"
+                            << "Constant with a 0 value was created, make sure connected input is "
+                               "optional.\n"
+                            << "If not check if the length of the data matches the shape. \n"
```

Comment:
```suggestion
                            << "Otherwise verify if the initializer contains a correct number of elements matching the initializer's shape. \n"
```